streaming/index.js

```diff
@@ -17,8 +18,13 @@ const WebSocket = require('ws');
 
 const environment = process.env.NODE_ENV || 'development';
 
+// Correctly detect .env/.env.production files based on whether we're running
```

Comment:
Is it possible to just pick one of these ways to run it, instead of handling either? ... or are there scenarios where we want to be running it different ways?

Reply:
basically if someone is in `mastodon/` and runs `node streaming/index.js` then it'll use `mastodon/.env` or `mastodon/.env.production`, but if you run from `mastodon/streaming` then it'll attempt to use `mastodon/streaming/.env` or `mastodon/streaming/.env.production` and silently fails (you get an database error)

Reply:
Fair enough. Is a pattern like this relevant? https://github.com/motdotla/dotenv/issues/238#issuecomment-348017037

Reply:
That'd pull in two additional files, and also search for .env in the user's home directory and other places: https://www.npmjs.com/package/find-config?activeTab=readme